lib/tasks/licenses.rake

```diff
@@ -1,10 +1,12 @@
 # frozen_string_literal: true
 
 namespace :licenses do  # rubocop:disable Metrics/BlockLength
-  report_path = Rails.root.join('doc/licenses/licenses.xml')
+  report_path = Rails.root.join('doc/licenses/licenses.xml').freeze
 
   desc 'Generates a report with the dependencies and their licenses'
   task :report do
+    require 'license_finder'
```

Comment:
You don't need the `require`, rails should make that job for you. I tried removing it and it works

Reply:
maybe in the past we had `require: false` in the Gemfile. I think it is good to have `require: false` in the Gemfile actually. But if we don't, then the `require 'license_finder'` is indeed not needed.